solutions/devsprint-leonardo-santos-3/FinanceAppTests/Services/Entities/AccountTests.swift

```diff
@@ -1 +1,20 @@
+import XCTest
+@testable import FinanceApp
 
+final class AccountTests: XCTestCase {
+    
+    // MARK: - Private Properties
+    
+    private let sut = Account(agency: "1111", account: "000011")
+    
+    // MARK: - Methods
+    
+    func test_formattedAccount_shouldReturnFormattedAccount() {
+        
+        // When
+        let formattedAccount = sut.formattedAccount()
```

Comment:
O que acha de colocarmos o nome de `expectedFormattedAccount`? Acredito que pro cenário de testes faz sentido com o que é esperado pra validação!

```suggestion
        let expectedFormattedAccount = sut.formattedAccount()
```

Reply:
opa @leocoout faz mais sentido mesmo.. Vou alterar aqui, muito obrigado!!